circuitbreaker.go

```diff
@@ -336,7 +357,9 @@ func (cb *Breaker) Call(circuit func() error, timeout time.Duration) error {
 
 // CallContext is same as Call but if the ctx is canceled after the circuit returned an error,
 // the error will not be marked as a failure because the call was canceled intentionally.
-func (cb *Breaker) CallContext(ctx context.Context, circuit func() error, timeout time.Duration) error {
+func (cb *Breaker) CallContext(
```

Comment:
If you're able to convince your editor not to do this, I'd rather not making meaningless formatting changes that make rebasing on any upstream changes harder.

Reply:
Done.